### ISSUE TITLE
"Popular in {your country}" categorization on onboarding

### ISSUE THREAD

**Issue by frosso** [Automattic/woocommerce-payments#1868]
## Description

The onboarding process categorizes the payment method in 2 categories:
- "Popular in {your country}"
- "Additional payment methods"

We need to determine how we can categorize the 2 and implement the solution

**Comment by RadoslavGeorgiev** [comment]
@frosso is this issue still applicable?

I see it mentioned in https://github.com/Automattic/woocommerce-payments/pull/1861, which I tested expecting to see something similar to what I saw in https://github.com/Automattic/woocommerce-payments/issues/1735, the issue describing it.

However, during setup I didn't see such a step, and after setup, I simply get redirected to WCPay's settings: https://d.pr/v/Pdy4WI.

With this in mind, could you either:

- Close the issue if no longer relevant.
- Add context (a dependency maybe?) of what needs to/will be implemented before we can work on this issue? Testing steps could be useful as well.

**Comment by frosso** [comment]
@RadoslavGeorgiev thanks for the ping!
I've been removed from this project for a while. In https://github.com/Automattic/woocommerce-payments/issues/1735 there's a screenshot that references this "categorization" functionality from older designs.

It looks like the onboarding functionality evolved a bit. It wasn't clear how this functionality would have worked at the time of writing and I don't see this functionality in the current designs, so I'm going to close this issue for now.
